lang/lvl_is_lex/lvl_is_lex.c

```diff
@@ -0,0 +1,197 @@
+/*  Permission is hereby granted, free of charge, to use and distribute  */
+/*  this software and its documentation without restriction, including   */
+/*  without limitation the rights to use, copy, modify, merge, publish,  */
+/*  distribute, sublicense, and/or sell copies of this work, and to      */
+/*  permit persons to whom this work is furnished to do so, subject to   */
+/*  the following conditions:                                            */
+/*   1. The code must retain the above copyright notice, this list of    */
+/*      conditions and the following disclaimer.                         */
+/*   2. Any modifications must be clearly marked as such.                */
+/*   3. Original authors' names are not deleted.                         */
+/*   4. The authors' names are not used to endorse or promote products   */
+/*      derived from this software without specific prior written        */
+/*      permission.                                                      */
+/*                                                                       */
+/*  CARNEGIE MELLON UNIVERSITY AND THE CONTRIBUTORS TO THIS WORK         */
+/*  DISCLAIM ALL WARRANTIES WITH REGARD TO THIS SOFTWARE, INCLUDING      */
+/*  ALL IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS, IN NO EVENT   */
+/*  SHALL CARNEGIE MELLON UNIVERSITY NOR THE CONTRIBUTORS BE LIABLE      */
+/*  FOR ANY SPECIAL, INDIRECT OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES    */
+/*  WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN   */
+/*  AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION,          */
+/*  ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF       */
+/*  THIS SOFTWARE.                                                       */
+/*                                                                       */
+/*************************************************************************/
+/*                                                                       */
+/*  is Lexical function                                            */
+/*                                                                       */
+/*************************************************************************/
+#include "flite.h"
+#include "cst_val.h"
+#include "cst_voice.h"
+#include "cst_lexicon.h"
+#include "cst_ffeatures.h"
+#include "lvl_is_lex.h"
+
+
+
+extern const int lvl_is_lex_entry[];
+extern const unsigned char lvl_is_lex_data[];
+extern const int lvl_is_lex_num_entries;
+extern const int lvl_is_lex_num_bytes;
+extern const char * const lvl_is_lex_phone_table[];
+extern const char * const lvl_is_lex_phones_huff_table[];
+extern const char * const lvl_is_lex_entries_huff_table[];
+
+
+cst_lexicon lvl_is_lex;
+cst_lts_rules lvl_is_lts_rules;
+extern const char * const lvl_is_lts_phone_table[];
+extern const char * const lvl_is_lts_letter_table[];
+extern const cst_lts_addr lvl_is_lts_letter_index[];
+extern const cst_lts_model lvl_is_lts_model[];
+
```

Comment:
hmm, looking at these: all of them are implecitely imported here, so we should really make sure that the ones we are declaring static aren't imported like this anywhere else. But this would result in `unresolved symbol` errors at the linker stage, so the compiler helps us in that regard.